tests/integration_tests/table_route/run.sh

```
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
```

Shellcheck (0.11.0)

[info] Not following: ./../_utils/test_prepare was not specified as input (see shellcheck -x).

(SC1091)